packages/api/src/router/widgets/media-transcoding.ts

```diff
@@ -0,0 +1,28 @@
+import { getIntegrationKindsByCategory } from "@homarr/definitions";
+import { mediaTranscodingRequestHandler } from "@homarr/request-handler/media-transcoding";
+import { z } from "@homarr/validation";
+
+import type { IntegrationAction } from "../../middlewares/integration";
+import { createOneIntegrationMiddleware } from "../../middlewares/integration";
+import { createTRPCRouter, publicProcedure } from "../../trpc";
+
+const createIndexerManagerIntegrationMiddleware = (action: IntegrationAction) =>
+  createOneIntegrationMiddleware(action, ...getIntegrationKindsByCategory("mediaTranscoding"));
+
+export const mediaTranscodingRouter = createTRPCRouter({
+  getDataAsync: publicProcedure
+    .input(z.object({ pageSize: z.number().min(0).max(100), pageOffset: z.number().min(1) }))
+    .unstable_concat(createIndexerManagerIntegrationMiddleware("query"))
+    .query(async ({ ctx, input }) => {
+      const innerHandler = mediaTranscodingRequestHandler.handler(ctx.integration, {
+        pageOffset: input.pageOffset,
+        pageSize: input.pageSize,
+      });
+      const { data } = await innerHandler.getCachedOrUpdatedDataAsync({ forceUpdate: false });
+
+      return {
+        integrationId: ctx.integration.id,
```

Comment:
We should return all data from integrations (except secrets), so it's easier to handle in the front end.
Mostly used so we can show what data belongs to what integration when there are multiple.

Reply:
At this time, we don't need to return all data - it's not needed. As soon as we do need it, we can return it